### PR TITLE
Updated README files to be more descriptive, updated gitignore

### PR DESCRIPTION
**Pull request by npasricha-cs**
Updated README files to be more descriptive. Added .DS_Store to .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
-## Cortex Certifai Examples 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Examples 
+This directory contains examples and tutorials illustrating how to use Certifai in a notebook environment and CLI.
 
+### Notebooks
+The *notebooks* folder contains Jupiter Lab notebooks which demonstrate how to use various functions of Certifai in a notebook environment. 
+
+### Tutorials
+The *tutorials* folder contains walkthroughs outside of a notebook environment which demonstrate various Certifai functionality.
```
Comment (cdraper-CS) [inline README.md:8]: I think we should include all of the tutorial material here, whether or not they are in a notebook

File: notebooks/README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
-## Cortex Certifai Tutorials 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Notebooks
+This directory contains *Jupyter notebooks* illustrating how to use Certifai in a notebook environment.
 
+### Azure ML Model Headers Demo
+
+ Azure Model headers demo example notebook highlights end-to-end process of running a Ceritifai Fairness evaluation by starting out with training, testing and deploying a sklearn model as a web service with token based Authentication on Azure, using Azure Containerzed Instance(ACI) and launcing a Certifai model scan on the deployed model in the same notebook. The exame notebook can be run either on local machine or on Azure hosted notebook environments.
```
Comment (cdraper-CS) [inline notebooks/README.md:6]: Ceritifai => Certifai

File: notebooks/README.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+
+### Bringing In Your Own Model to Certifai
+
+ Bringing In Your Own Model to Certifai example demonstrates how create a scan in Certifai using your own model. In this example the model being used is Logistic Regression. We will show how to configure the model to be used in Certifai, and run a scan on it with key evaluation metrics:
```
Comment (cdraper-CS) [inline notebooks/README.md:10]: how to

File: notebooks/README.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
-## Cortex Certifai Tutorials 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Notebooks
+This directory contains *Jupyter notebooks* illustrating how to use Certifai in a notebook environment.
```
Comment (cdraper-CS) [inline notebooks/README.md:2]: Are we going to duplicate this information in the docs and here? Can't we just refer to docs for a description?  Or vice versa? @smichalski-cs

File: notebooks/azureml_model_headers_demo/.gitignore [hunk 5]
```diff
@@ -0,0 +1,5 @@
+__pycache__
+*.pkl
+.ipynb_checkpoints
+.idea
+.DS_Store
```
Comment (cdraper-CS) [inline notebooks/azureml_model_headers_demo/.gitignore:5]: I'd recommend just adding this .gitignore at the top level

File: tutorials/README.md [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
+
+Prerequisites:
+
+* Certifai Toolkit 
+* Scan Definition file (from part 1)
+* A model of your choice
+* Train and Predict functions for the model
+
+### Documentation 
 These tutorials have docs to be followed along, which can be found at: https://cognitivescale.github.io/cortex-certifai/docs/about
```
Comment (cdraper-CS) [inline tutorials/README.md:18]: The `about` link seems an odd choice for getting people to the tutorial docs.

File: tutorials/README.md [hunk 7]
```diff
@@ -1,5 +1,19 @@
-## Cortex Certifai Tutorials 
-This directory contains the self contained tutorials which walk through various parts of Certifai. 
+# Cortex Certifai Tutorials 
+This directory contains self contained tutorials which walk through various parts of Certifai *outside* of a notebook environment.
 
+### Bringing In Your Own Model CLI
+
+This tutorial continues from part 1 located in the *notebooks* folder.
```
Comment (cdraper-CS) [inline tutorials/README.md:6]: Reminder to adjust this if you move all the tutorials under the one folder

File: tutorials/README.md (hunk 6 above)
Comment (npasricha-cs) [reply tutorials/README.md:18]: It is a placeholder as there is no link to an overall tutorials or notebooks page in the docs

File: notebooks/README.md [hunk 8]
```diff
@@ -1,5 +1,10 @@
...
 
+### Azure ML Model Headers Demo
+
+ Azure Model headers demo example notebook highlights end-to-end process of running a Certifai Fairness evaluation by starting out with training, testing and deploying a sklearn model as a web service with token based Authentication on Azure, using Azure Containerized Instance(ACI) and launcing a Certifai model scan on the deployed model in the same notebook. The example notebook can be run either on local machine or on Azure hosted notebook environments.
```
Comment (cdraper-CS) [inline notebooks/README.md:6]: launcing => launching

File: README.md (hunk 1 above)
Comment (cdraper-CS) [inline README.md:8]: Drop "outside of a notebook environment".

Comment (npasricha-cs) [reply README.md:8]: Done, thanks!

**Comment by npasricha-cs** [comment]
@akumar-cs @smichalski-cs @skottaram-cs Could you take a look please? Want to get this merged in

File: notebooks/README.md (hunk 4 above)
Comment (smichalski-cs) [reply notebooks/README.md:2]: I think we should keep the descriptions in the docs and I can link to the examples repo there

File: README.md [hunk 9]
```diff
@@ -1,5 +1,12 @@
-## Cortex Certifai Examples 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Examples 
+This directory contains examples and tutorials illustrating how to use Certifai in a notebook environment and CLI.
```
Comment (smichalski-cs) [inline README.md:2]: Change to: This directory contains example files, notebooks, and tutorials that will help you get started using Certifai tools.

File: notebooks/README.md (hunk 4 above)
Comment (smichalski-cs) [reply notebooks/README.md:2]: Just link to the description page in the docs. I will link to the repo from the docs

File: tutorials/README.md [hunk 10]
```diff
@@ -1,5 +1,16 @@
-## Cortex Certifai Tutorials 
-This directory contains the self contained tutorials which walk through various parts of Certifai. 
+# Cortex Certifai Tutorials 
+This directory contains self contained tutorials which walk through basic parts of Certifai funcitonality
```
Comment (smichalski-cs) [inline tutorials/README.md:2]: remove "self-contained" ; change "which" to "that"; remove "parts of"

File: notebooks/README.md [hunk 11]
```diff
@@ -1,5 +1,10 @@
-## Cortex Certifai Tutorials 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Notebooks
+This directory contains *Jupyter notebooks* illustrating how to use Certifai in a notebook environment.
 
+### Azure ML Model Headers Demo
+
+ Azure Model headers demo example notebook highlights end-to-end process of running a Certifai Fairness evaluation by starting out with training, testing and deploying a sklearn model as a web service with token based Authentication on Azure, using Azure Containerized Instance(ACI) and launching a Certifai model scan on the deployed model in the same notebook. The example notebook can be run either on local machine or on Azure hosted notebook environments.
+
```
Comment (smichalski-cs) [inline notebooks/README.md:7]: token-based and Azure-hosted (you need hyphens)

File: tutorials/README.md [hunk 12]
```diff
@@ -1,5 +1,16 @@
-## Cortex Certifai Tutorials 
-This directory contains the self contained tutorials which walk through various parts of Certifai. 
+# Cortex Certifai Tutorials 
+This directory contains self contained tutorials which walk through basic parts of Certifai funcitonality 
 
+### Bringing In Your Own Model to Certifai Part 1
+
+ Bringing In Your Own Model to Certifai example demonstrates how to create a scan in Certifai using your own model. In this example the model being used is Logistic Regression. We will show how to configure the model to be used in Certifai, and run a scan on it with key evaluation metrics:
+
```
Comment (smichalski-cs) [inline tutorials/README.md:7]: Remove "We will". change to: This tutorial shows . . .; remove "on it"

File: tutorials/README.md [hunk 13]
```diff
@@ -1,5 +1,16 @@
...
+ * Fairness
+ * Explainability
+ * Robustness
+
+The end result will be a scan definition file which contains the meta data from our scan, which can be used for running scans in the CLI and in Part 2 of this tutorial.
```
Comment (smichalski-cs) [inline tutorials/README.md:12]: change to . . . that contains the metadata from your scan, which is used in part two of this tutorial."

**Comment by npasricha-cs** [comment]
Updated with latest requested changes @cdraper-CS @smichalski-cs

**Pull request merged**